두-정수-사이의-합.js

```diff
@@ -0,0 +1,22 @@
+function solution(a, b) {
+    const num = [];
+    
+    let answer = 0;
```

Comment:
변수 선언끼리는 줄 바꿈을 해주지 않아요!
```suggestion
    const num = [];
    let answer = 0;
```